spec/services/unpublish_service_spec.rb

```diff
@@ -67,6 +73,21 @@
           .to raise_error RuntimeError, "Publishing API does not support unpublishing while there is a draft"
       end
     end
+
+    context "when the edition is already withdrawn" do
+      it "updates the public explanation" do
+        edition = create(:edition, :withdrawn)
+        expect { UnpublishService.new.withdraw(edition, public_explanation, user) }
+          .to change { edition.reload.status.details.public_explanation }
+          .to(public_explanation)
+      end
+
+      it "maintains the withdrawn timestamp" do
```

Comment:
I wonder if this is actually testing anything, as it's very possible the system time hasn't advanced enough to create any potential difference to test against; probably we need to specify a time that's definitely older in order to test this.

Reply:
Well yeah time will always have moved forward by then. I can make it clearer for you though.